repo/packages/W/weavescope-probe/0/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "object",
+  "description": "Properties of Weave Scope probe package",
+  "properties": {
+    "weavescope": {
```

Comment:
currently your services are deployed with a `-default` as part of the name which probably frustrates dns look up of the service.

if you have a property "service" peer to "weavescope" here, then have a sub property of "name" with a value of "weavescope". it will fix that.

I recently needed this for a KDC service.  here is the example:  https://github.com/mesosphere/universe/blob/kerberos-hdfs/repo/packages/K/kdc/0/config.json#L5-L9